charts/kube-prometheus-stack/templates/grafana/configmap-dashboards.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if and .Values.grafana.enabled .Values.grafana.defaultDashboardsEnabled }}
-{{- $files := .Files.Glob "dashboards/*.json" }}
+{{- $files := .Files.Glob "dashboards-1.14/*.json" }}
```

Comment:
reviewers, PTAL if this is correct.

Reply:
I don't see any previous reference to `dashboards-1.14`. Was it never used then? :thinking: